jbmc/src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -650,20 +650,20 @@ static irep_idt get_if_cmp_operator(const irep_idt &stmt)
 /// Build a member exprt for accessing a specific field that may come from a
 /// base class.
 /// \param pointer: The expression to access the field on.
-/// \param fieldref: A getfield/setfield instruction produced by the bytecode
+/// \param field_instruction: A getfield/setfield instruction produced by the bytecode
 /// parser.
 /// \param ns: Global namespace
 /// \return A member expression accessing the field, through base class
 /// components if necessary.
 static member_exprt
-to_member(const exprt &pointer, const exprt &fieldref, const namespacet &ns)
+to_member(const exprt &pointer, const exprt &field_instruction, const namespacet &ns)
```

Comment:
Can we make the type of field_instruction more precise? Is it a `member_designatort`?
There seems to be some implicit assumption about the expression (for instance it has an `ID_class`).

Reply:
@smowton points out here `field_instruction` is a misleading name since it isn't an instruction - suggests calling it just `field_reference`, would you be happy with that @romainbrenguier (since you objected to `fieldref`)?)